CONTRIBUTING.md

```diff
@@ -26,6 +26,15 @@ and review all changes we merge to master.
 We appreciate any contributions, so please use the [Forking Workflow](https://www.atlassian.com/git/tutorials/comparing-workflows/forking-workflow)
 and send us `Merge Requests`. 
 
+### Patch files generation
+
+Some components may use git patch files to add functionality or fix bugs, such
+as linux kernel or u-boot, please use below command to generate patch:
```

Comment:
"... linux kernel or u-boot. Please use the command below to generate such patches:"

Reply:
Ack & changed.